cgo/libclang_config.go

```diff
@@ -1,14 +1,16 @@
-// +build !byollvm
-// +build !llvm12
+//go:build !byollvm && !llvm12
+// +build !byollvm,!llvm12
 
 package cgo
 
 /*
-#cgo linux   CFLAGS:  -I/usr/lib/llvm-11/include
-#cgo darwin  CFLAGS:  -I/usr/local/opt/llvm@11/include
-#cgo freebsd CFLAGS:  -I/usr/local/llvm11/include
-#cgo linux   LDFLAGS: -L/usr/lib/llvm-11/lib -lclang
-#cgo darwin  LDFLAGS: -L/usr/local/opt/llvm@11/lib -lclang -lffi
-#cgo freebsd LDFLAGS: -L/usr/local/llvm11/lib -lclang
+#cgo linux			CFLAGS:  -I/usr/lib/llvm-11/include
```

Comment:
This may be a bug of github frontend. In my local goland, this format is well:

![image](https://user-images.githubusercontent.com/3314335/147754902-97188a63-ce14-40b9-b8fb-25a33b2be046.png)

Reply:
It looks like Goland doesn't show removed lines. The GitHub diff is correct. You have modified the other lines (linux, freebsd) as well.

Can you either undo those modifications or make sure the color character is aligned for all `#cgo` lines?

Reply:
I edited these lines online on github, and this should be fine now.